ios/PurchasesHybridCommon/PurchasesHybridCommonUI/PaywallProxy.swift

```diff
@@ -90,6 +97,25 @@ import UIKit
                                    paywallResultHandler: paywallResultHandler)
     }
 
+    @objc
+    public func presentPaywall(options: [String:Any],
```

Comment:
Adding parameters to this function was getting very complex, since it involves adding all combinations, due to `@objc` not allowing for optional parameters. I instead created this one, that even though the options are not typed, I think it cleans and will clean up the future code.